apps/ui/src/components/Modal/Strategies.vue

```diff
@@ -0,0 +1,123 @@
+<script lang="ts">
```

Comment:
What's the rational of having this separated from the other script tag ?

Reply:
This way this ref is shared across all instances of ModalStrategies.